KA-Flocking/Assets/Scripts/FlockAgent.cs

```diff
@@ -22,12 +22,16 @@ void Start()
         rb = GetComponent<Rigidbody>();
     }
 
-    public void Move(Vector3 velocity)
+    public void Move(Vector3 acceleration)
     {
-        transform.position += velocity * Time.deltaTime;
-        if (velocity != Vector3.zero)
+        
+        acceleration.Normalize();
```

Comment:
acceleration is already normalized before call (in Flock row 50). It is however multiplied with maxspeed afterwards